pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -226,30 +228,32 @@ void processArguments() throws Exception {
                 description = "Path to the main Python file for the function (if the function is written in Python)",
                 listConverter = StringConverter.class)
         protected String pyFile;
-        @Parameter(names = "--inputs", description = "The function's input topic or topics (multiple topics can be specified as a comma-separated list)")
+        @Parameter(names = { "-i",
+                "--inputs" }, description = "The function's input topic or topics (multiple topics can be specified as a comma-separated list)")
         protected String inputs;
         // for backwards compatibility purposes
         @Parameter(names = "--topicsPattern", description = "TopicsPattern to consume from list of topics under a namespace that match the pattern. [--input] and [--topic-pattern] are mutually exclusive. Add SerDe class name for a pattern in --custom-serde-inputs (supported for java fun only)", hidden = true)
         protected String DEPRECATED_topicsPattern;
         @Parameter(names = "--topics-pattern", description = "The topic pattern to consume from list of topics under a namespace that match the pattern. [--input] and [--topic-pattern] are mutually exclusive. Add SerDe class name for a pattern in --custom-serde-inputs (supported for java fun only)")
         protected String topicsPattern;
-        @Parameter(names = "--output", description = "The function's output topic")
+
+        @Parameter(names = {"-o", "--output"}, description = "The function's output topic")
         protected String output;
         // for backwards compatibility purposes
         @Parameter(names = "--logTopic", description = "The topic to which the function's logs are produced", hidden = true)
         protected String DEPRECATED_logTopic;
         @Parameter(names = "--log-topic", description = "The topic to which the function's logs are produced")
         protected String logTopic;
+
+        @Parameter(names = {"-st", "--schema-type"}, description = "The builtin schema type or custom schema class name to be used for messages output by the function")
+        protected String schemaTypeOrClassName = "";
+
         // for backwards compatibility purposes
         @Parameter(names = "--customSerdeInputs", description = "The map of input topics to SerDe class names (as a JSON string)", hidden = true)
```

Comment:
No, these were left there also in the other change, as hidden, to not break compatibility

Reply:
For instance --custom-serde-inputs(and other changes) are being deleted by this pr. That doesn't sound right?

Reply:
Oh, right, the thing is that Jerry's PR moved from `--customSerdeInput` to `--custom-serde-inputs`, leaving both for compatibility and  `--customSerdeInput` as hidden. 

Since with this PR I'm deprecating  `--customSerdeInput`, there's no need to keep `--custom-serde-inputs` because it was never released.

Reply:
makes sense